pmm/pmm2-ui-tests.groovy

```diff
@@ -168,8 +160,10 @@ pipeline {
                         expression { env.CLIENT_INSTANCE == "no" }
                     }
                     steps {
-                        installAWSv2()
-                        withCredentials([aws(accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY')]) {
+                        withCredentials([aws(accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                                         credentialsId: 'AMI/OVF',
+                                         secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'
+                                         )]) {
                             sh """
                                 aws ecr-public get-login-password --region us-east-1 | docker login -u AWS --password-stdin public.ecr.aws/e7j3v3n0
```

Comment:
Is this OK or should it be docker.io as well?

Reply:
I removed it too